src/app/(pages)/(TwoColumn)/article/page.jsx

```diff
@@ -9,6 +9,7 @@ import PageTitle from '@/components/common/PageTitle'
 import TagArea from '@/components/common/TagArea'
 import TagKeyword from '@/components/common/TagKeyword'
 
+export const dynamic = 'force-static';
```

Comment:
same above

Reply:
I believe this page should work on some sonditions like `page` or `tag_id`,
Although those conditions are not able to get from server-side.
Apparently using `force-dynamic` is injecting null into `searchParams` param ([ref](https://github.com/vercel/next.js/issues/50999#issuecomment-1727941641)).
So my suggestion is how about using client-side fetch for this page?

Reply:
I might be wrong but I think that won't work too. 
Because if you look at [reminder/page.jsx](https://github.com/diverta/front_next_media/blob/main/src/app/(pages)/(SigleColumn)/reminder/page.jsx#L11), this is an example of using `searchParams` with `use-client` 

But this page still seems to fail at `npm run build`. with or without the `force-static` constraint

Please check once

Reply:
This page is successfully generated by build command.
So this conversation is a bit stale.
Closing.